hr_timesheet_report_rounded/report/hr_timesheet_report.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018-2020 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models
+from odoo.osv.expression import TRUE_DOMAIN
+from odoo.tools.safe_eval import safe_eval
+
+
+class HrTimesheetReportEntry(models.TransientModel):
+    _inherit = "hr.timesheet.report.entry"
+
+    def _compute_total_unit_amount(self):
```

Comment:
I would suggest making this behavior configurable with being off by default so that this module can coexist with others in tests.

Reply:
@alexey-pelykh thank you for the feedback! 
The idea behind this module being installed automatically is that it gives 100% insurance of the correct time being used in reports.  Making this optional may lead to a situation where incorrect values are added in the report when someone forgets to enable this setting.
So let's update the rebel modules instead.

Reply:
I meant system setting or that sort of stuff. However since sale_timesheet_rounded is also non-configurable, it's fine.